enterprise/server/test/webdriver/invocation/invocation_test.go

```diff
@@ -2,18 +2,90 @@ package invocation_test
 
 import (
 	"context"
+	"flag"
 	"testing"
 
 	"github.com/buildbuddy-io/buildbuddy/enterprise/server/testutil/buildbuddy_enterprise"
+	"github.com/buildbuddy-io/buildbuddy/server/testutil/app"
 	"github.com/buildbuddy-io/buildbuddy/server/testutil/testbazel"
 	"github.com/buildbuddy-io/buildbuddy/server/testutil/webtester"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 )
 
-func TestAuthenticatedInvocation_LogUploadEnabled(t *testing.T) {
+var (
+	targetStr   = flag.String("webdriver_target", "local", "Target that should be tested by the webdriver.")
+	devSSOSlug  = flag.String("webdriver_dev_slug", "", "Dev SSO slug to login.")
+	prodSSOSlug = flag.String("webdriver_prod_slug", "", "Prod SSO slug to login.")
+)
+
+type webdriverTarget interface {
```

Comment:
Can we move these structs and flags to a shared package, maybe `server/testutil/testtarget` or `server/testutil/webtest_target` ? (doesn't need to be under enterprise/, since we might want to have some library functions which depend on the Target, but can be used with OSS webdriver tests). That way when we add more webdriver tests we don't need need to duplicate these

Reply:
Moved it under /enterprise because OSS doesn't support auth. If we end up needing these drivers in the OSS tests, we can refactor this later to move the interface to OSS, but keep drivers with auth in /enterprise